lib/makersbnb.rb

```diff
@@ -0,0 +1,29 @@
+require 'pg'
+
+class MakersBnb_Listings
+
+  def self.create_space(space_name:, price:, description:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'makers_bnb_test')
+    else
+      connection = PG.connect(dbname: 'makers_bnb')
+    end
+    connection.exec_params("INSERT INTO spaces (name, price, description) VALUES($1, $2, $3) RETURNING name, price, description;", [space_name, price, description]
+    )
+  end
+
+  def self.exist?(space_name:)
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'makers_bnb_test')
+    else
+      connection = PG.connect(dbname: 'makers_bnb')
+    end
+    result = connection.exec_params("SELECT * FROM spaces WHERE name = $1;",[space_name])
+    @unsuccessful = !result.to_a.empty?
```

Comment:
Can you please explain what is being done here? I think we have something similar and used result.count.zero? Which returns true if the sql query being run has no results.

Reply:
Yeah pretty similar I think, we're just checking if the space name has already been taken or if it's free

Reply:
yes agree result.count.zero? would improve readability